frontend/src/components/ResetModal/ResetModal.test.tsx

```diff
@@ -63,12 +63,14 @@ describe('ResetModal', () => {
     ]);
   });
 
-  it('should redirect back to course selector if cancel button is clicked', async () => {
-    const dummyNavigate = vi.fn();
-    useNavigateMock.mockReturnValue(dummyNavigate);
+  // TODO: Disabled since it now takes a onCancel to be reusable
```

Comment:
Since we are changing the interface of `ResetModal`,  rather than commenting this out, could we change the test? We can have a test case where we check if it calls the `onCancel` callback passed when clicking the cancel button.

Also, if you want, can add another test to see if `onOk` callback passed is called when clicking the confirm button. 

Though let me know if you want me to this bit or not if you're confused on writing FE tests. I can write the tests if you want :D

Reply:
goooood idea!!!

Reply:
oki done <3